privacyidea/models.py

```diff
@@ -1389,11 +1389,16 @@ class Policy(TimestampMethodsMixin, db.Model):
     # If there are multiple matching policies, choose the one
     # with the lowest priority number. We choose 1 to be the default priotity.
     priority = db.Column(db.Integer, default=1, nullable=False)
+    extra_conditions = db.relationship("PolicyExtraCondition",
```

Comment:
I think we can rename this to ``PolicyCondition`` so that one day all conditions are stored there.

Reply:
Done in f0be606